admin/debug.log

```diff
@@ -0,0 +1 @@
+[0923/141044.621:ERROR:directory_reader_win.cc(43)] FindFirstFile: System nie mo�e odnale�� okre�lonej �cie�ki. (0x3)
```

Comment:
Please add all logs into gitignore. BTW why we have logs in repository?

Reply:
Oversight :P